resources/assets/js/features/search/components/Result/Filter.js

```diff
@@ -82,12 +82,12 @@ class Filter extends React.Component {
         setUrl(setFilter({rating: e.target.value}));
     }
 
-    transferChange(e) {
-        setUrl(setFilter({transfer: e.target.value}));
+    transfersChange(e) {
+        setUrl(setFilter({transfers: e.target.value}));
```

Comment:
default value should be null or 0 or empty string at least, but not string "false"